ERCS/erc-5564.md

```diff
@@ -156,7 +157,7 @@ The new address format for the stealth meta-address extends the chain specific a
 Thus, a stealth meta-address on Ethereum has the following format:
 
 ```
-st:eth:0x<spendingKey><viewingKey>
+st:eth:0x<spendingPubKey><viewingPubKey>
 ``` 
 
 Stealth meta-addresses may be managed by the user and/or registered within a publicly available `Registry` contract, as delineated in [ERC-6538](./eip-6538). This provides users with a centralized location for identifying stealth meta-addresses associated with other individuals while simultaneously enabling recipients to express their openness to engage via stealth addresses.
```

Comment:
ERC-6538 is Stagnant, and mentioning it here prevents moving this proposal out of Draft. Please either re-activate that proposal, or remove the reference here.

Reply:
We have submitted a PR at https://github.com/ethereum/ERCs/pull/259 to advance ERC-6538 to the review stage, currently waiting for editors' review.

Reply:
https://github.com/ethereum/ERCs/pull/259 has been merged in!